ui/src/app/views/dataset-page.tsx

```diff
@@ -504,12 +538,12 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
                   </div>
                 </div>
                 <div style={{width: '40%'}}>
-                  <div style={{...styles.selectBoxHeader, display: 'flex'}}>
+                  <div style={styles.selectBoxHeader}>
```

Comment:
similar questions on styles

Reply:
moved the display flex into selectBoxHeader since it's used in all selectBoxHeaders now